wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -116,6 +117,30 @@ def convert_mapped_fields(self, record_fields_dict, mapped_fields_dict) -> dict:
         }
         return mapped_fields_dict
 
+    def update_model_m2m_fields(self, instance, field_name, value) -> None:
+        m2m_field = getattr(instance, field_name)
+        for m2m_value in value:
+            m2m_field.add(m2m_value)
+
+    def check_field_is_m2m(self, model, data: dict):
+        for field_name, value in data.items():
+            field_type = type(
+                model._meta.get_field(field_name)
+            )  # ie. django.db.models.fields.CharField
+            # If this field type is a subclass of a known Wagtail Tag, or a Django m2m field
+            # We need to loop through all the values and add them to the m2m-style field.
+            if issubclass(
+                    field_type,
+                    (
+                            TaggableManager,
+                            ClusterTaggableManager,
+                            models.ManyToManyField,
+                    ),
+            ):
+                yield field_name, value, True
```

Comment:
Could we just store the `issubclass(...)` in a variable and yield it alongside `field_name` and `value` instead of needing the `if`? A nitpick, just slightly neater.